src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,49 @@
 package mate.academy;
 
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    public static final Injector injector = Injector.getInstance("mate.academy");
+    public static final MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
+    public static final MovieSessionService movieSessionService =
+            (MovieSessionService) injector.getInstance(MovieSessionService.class);
+    public static final CinemaHallService cinemaHallService =
+            (CinemaHallService) injector.getInstance(CinemaHallService.class);
 
+    public static void main(String[] args) {
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
+
+        Movie movie = new Movie("movie1");
+        movie.setDescription("movie");
+        movieService.add(movie);
+        System.out.println(movieService.get(movie.getId()));
```

Comment:
The creation of the `movie` object and its subsequent operations seem redundant given the earlier creation of `fastAndFurious`. Consider removing these lines if they are not necessary for demonstrating specific functionality.